.gitignore

```diff
@@ -1,5 +1,5 @@
 # Build files
-build/
+build*/
```

Comment:
This allows you to have multiple builds which allows you to setup clion to literally just select what chip you are building for instead of having to edit the cmake args every time you want to build for a different chip. Really useful when constantly switching between chips. This is a good thing.

Reply:
How would you like to write some documentation explaining how to do this?

Reply:
Should probably just be a comment on the website, likely under the running code page. Nothing has fundamentally changed, there are just more options now for the build folder.

Reply:
Added as a Jira issue.